### ISSUE TITLE
[QUESTION] Yubikey NFC + OTP

### ISSUE THREAD

**Issue by paxis2** [PhilippC/keepass2android#2681]
How do I get Yubikey to work for OTP using NFC? I have set this up on PC and it works. I am on Keepass2Android version 1.11-r0. When I open the database, it says it requires the Yubiclip app.

**Comment by schlomie** [comment]
Install [ykDroid](https://f-droid.org/en/packages/net.pp3345.ykdroid/) from FDroid.

**Comment by paxis2** [comment]
> Install [ykDroid](https://f-droid.org/en/packages/net.pp3345.ykdroid/) from FDroid.

I already have this installed. Are there specific settings required? When I tap Yubikey in Keepass2android, it just shows me options to open in different browsers.

**Comment by schlomie** [comment]
When you are unlocking your database, it lets you select the master key type.  This dropdown defaults to `Password only`.  To make use of ykDroid, select `Password + Challenge-Response for Keepass XC` (or whichever one is suitable or your use case).

Once you enter your password and click unlock, it will then prompt you to touch your yubikey to the back of your phone.

**Comment by paxis2** [comment]
![20241010_121853](https://github.com/user-attachments/assets/4d7e85d8-e9d7-4c50-8184-48b373f90732)

Yup, that is what I did. When I tap to the back of the phone for NFC, this pops up. I inserted Yubikey into my phone via USB and that works fine, so it is something to do with NFC. For background, I also have Yubico Authenticator on my phone for other accounts.

**Comment by schlomie** [comment]
Interesting.   You have the Yubikey NEO?  I have a YK5 - maybe that is the difference.  I dunno.  Sorry!

**Comment by paxis2** [comment]
Yes I use a Yubikey 5 NFC. Did you have Yubico Authenticator installed? Wonder if that is causing the issue for me.

**Comment by zkzkzk2015** [comment]
I am using Yubikey Authenticator and Yubikey 5 NFC. It is working flawlessly at least for a year on my OP 9 Android 14.
